crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -195,12 +195,11 @@ impl BaseRoomInfo {
                 let mut o_ev = o_ev.clone();
                 o_ev.content.set_created_ts_if_none(o_ev.origin_server_ts);
 
-                let Some(owned_user_id) = get_user_id_for_state_key(m.state_key()) else {
```

Comment:
this changes behavior, are you sure this new behavior is correct? does Ruma do the complicated parsing now? cc @toger5

Reply:
Yes it does, see https://github.com/ruma/ruma/pull/1889.

Reply:
Cool cool